serverless.yml

```diff
@@ -11,7 +11,7 @@
 #
 # Happy Coding!
 
-service: partyshe-first-app
+service: partyshe-third-app-service
```

Comment:
This MUST be the same service. 

You have ONE application

Reply:
Would you like to create new application per feature?

Reply:
Thank you, Sergey. I understand. I've deleted all services and buckets and played with deployment. 
If naming is not critical and if you don't mind, I want to keep current version.